docs/core/deploying/index.md

```diff
@@ -219,58 +218,192 @@ A complete sample `project.json` file appears later in this section.
+
+    ```console
+    dotnet build -r win10-x64
+    dotnet build -r osx.10.10-x64
+    ```
+
+6. After you've debugged and tested the program, you can create the files to be deployed with your app for each platform that it targets by using the `dotnet publish` command for both target platforms as follows:
+
+   ```console
+   dotnet publish -c release -r win10-x64
+   dotnet publish -c release -r osx.10.10-x64
+   ```
+This creates a release (rather than a debug) version of your app for each target platform. The resulting files are placed in a subdirectory named `publish` that is in a subdirectory of your project's `.\bin\release\netstandard1.6\<runtime_identifier>` subdirectory. Note that each subdirectory contains the complete set of files (both your app files and all .NET Core files) needed to launch your app.
+
+7. Along with your application's files, the publishing process emits a program database (.pdb) file that contains debugging information about your app. The file is useful primarily for debugging exceptions; you can choose not to package it with your application's files.
+
+The published files can be deployed in any way you'd like. For example, you can package them in a zip file, use a simple `copy` command, or deploy them with any installation package of your choice. Before packaging and deploying your app, you can also use `crossgen` to convert it to native code. For more information, see the [Native Image Generation](#crossgen) section.
+
+The following is the complete `project.json` file for this project.
+
+```json
+   {
+     "version": "1.0.0-*",
+     "buildOptions": {
+       "debugType": "portable",
+       "emitEntryPoint": true
+     },
+     "dependencies": {
+       "NETStandard.Library": "1.6.0",
+       "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2",
+       "Microsoft.NETCore.DotNetHostPolicy":  "1.0.1"
+     },
+     "frameworks": {
+       "netstandard1.6": { }
+     },
+     "runtimes": {
+       "win10-x64": {},
+       "osx.10.10-x64": {}
+     }
+   }
+```
+
 ## <a name="crossgen"></a> Native Image Generation ##
 
 NET Core uses a just in time (JIT) compiler that stores application code in an intermediate format and compiles it to native code at runtime.  To increase startup performance, the shared framework is pre-compiled using a tool called `crossgen`.  To improve performance of your application, you can use the same tool on your application's binaries.  Its performance impact is more noticeable when deploying a standalone application, since the entire framework is part of the application. Crossgen must be run on a machine of the same platform type that you are targeting, but need not be done on the same machine, unlike ngen for the desktop framework.  If you are producing a platform-specific installer for your application, we recomend that you crossgen as part of the installer build process.
```

Comment:
You use "standalone" here but "self-contained" in the document. We should only have one way to call the thing. :smiley: